src/AqBanking/Command/RenderContextFileToSimpleXMLCommand.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace AqBanking\Command;
+
+use AqBanking\ContextFile;
+
+class RenderContextFileToSimpleXMLCommand extends AbstractCommand
+{
+    /**
+     * @param ContextFile $contextFile
+     * @return \DOMDocument
+     * @throws \RuntimeException
+     */
+    public function execute(ContextFile $contextFile)
+    {
+        $shellCommand =
+            $this->pathToAqBankingCLIBinary
+            . ' export'
+            . ' --ctxfile=' . escapeshellcmd($contextFile->getPath())
+            . ' --exporter=xmldb';
+
+        $result = $this->getShellCommandExecutor()->execute($shellCommand);
+
+        if ($result->getReturnVar() !== 0) {
+            throw new \RuntimeException(
+                'AqBanking exited with errors: ' . PHP_EOL
+                . implode(PHP_EOL, $result->getErrors())
+            );
+        }
+
+        return new \SimpleXMLElement(implode(PHP_EOL, $result->getOutput()));
```

Comment:
I am not sure of https://github.com/Mestrona/aqbanking-php/blob/master/src/AqBanking/Command/RenderContextFileToXMLCommand.php should return DomDocument and another command simpleXML ... we might add a option switch to determine the output format, what do you think?

Reply:
Yes sure - I didn't want to make too many changes to your existing logic.

Reply:
is suggest to add `$returnSimpleXml = false` as a parameter and it will stay compatible :)